docs/README.MD

```diff
@@ -0,0 +1,3 @@
+# Documentation
+
+Please visit [github pages](https://project-eutopia.github.io/vega/).
```

Comment:
I noticed that we already link to this in the readme: https://github.com/project-eutopia/vega/blob/master/README.md#vega. Maybe it's not prominent enough though? I would be okay with you adding a new sub-heading that specifically just links to this documentation, but this `docs` folder is just for the Doxygen output (see HTML_OUTPUT value of the Doxyfile), so we shouldn't manually add any files to this folder.

Reply:
@project-eutopia Okey.
Maybe it's a good idea to add md5 files to .gitignore?

Reply:
The documentation in the `docs` folder is what is deployed to the Github pages you linked to here, so it should not be in the .gitignore.

Reply:
@project-eutopia I think files like [this](https://github.com/project-eutopia/vega/blob/master/docs/age_8h__incl.md5) are useless.

Reply:
The entire `docs` folder is autogenerated from the codebase's Doxygen comments. The generated map and md5 files have a specific purpose: https://sourceforge.net/p/doxygen/mailman/message/10983184/. If you still think they are useless I would recommend filing a bug with their repository: https://github.com/doxygen/doxygen/issues.

No one should really be looking at the autogenerated raw files in that directory, just the documentation that is deployed from the `docs` folder at https://project-eutopia.github.io/vega/.

Regardless, such changes to the documentation should be done in a separate PR, because the stated purpose of this PR is just to add an code example.

Reply:
Could you either remove this file from the pull request or update the main README.md file to make the hyperlink to the documentation more prominent like I suggested above? Thanks